platform/linux-generic/odp_dma.c

```diff
@@ -735,10 +840,12 @@ int odp_dma_transfer_done(odp_dma_t dma, odp_dma_transfer_id_t transfer_id,
 	}
 
 	if (result) {
```

Comment:
Also here the API states: `The call outputs optionally transfer results on a non-zero return value.` Should this be changed to exclude negative values?